### PR TITLE
ADEN-4318 Add unit tests for recovery disabling

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?php
 
 class ARecoveryModule {
+	const disabledMessage = PHP_EOL . '<!-- Recovery disabled. -->' . PHP_EOL;
```
Comment (WikiaTech) [inline extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php]: MINOR Rename this constant "disabledMessage" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)

File: extensions/wikia/ARecoveryEngine/tests/ARecoveryModuleTest.php [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<?php
+
+
+class ARecoveryModuleTest extends WikiaBaseTest {
+	const bootstrapCode = 'BOOTSTRAP_CODE';
```
Comment (WikiaTech) [inline extensions/wikia/ARecoveryEngine/tests/ARecoveryModuleTest.php]: MINOR Rename this constant "bootstrapCode" to match the regular expression ^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS115)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by jacekjursza** [comment]
LGTM

**Comment by jacekjursza** [comment]
LGTM

**Pull request merged**